tests/system/action/user/test_save_saml_account.py

```diff
@@ -509,34 +510,115 @@ def setUp(self) -> None:
-                    "external_id": "Landtag",
-                    "external_group_id": "Delegates",
-                },
+                "meeting_mappers": [
+                    {
+                        "name": "works",
+                        "external_id": "Landtag",
+                        "conditions": [
+                            {"attribute": "member_number", "condition": "LV_.*"},
+                            {
+                                "attribute": "email",
+                                "condition": "[\\w\\.]+@([\\w-]+\\.)+[\\w]{2,4}",
+                            },
+                        ],
+                        "mappings": {
+                            "comment": {
+                                "attribute": "idp_commentary",
+                                "default": "Vote weight, groups and structure levels set via SSO.",
+                            },
+                            "number": {"attribute": "participant_number"},
+                            "structure_levels": {
+                                "attribute": "structure",
+                                "default": "structure1",
+                            },
+                            "groups": {
+                                "attribute": "idp_group_attribute",
+                                "default": "not_a_group",
+                            },
+                            "vote_weight": {"attribute": "vw", "default": "1.000000"},
+                            "present": {"attribute": "presence", "default": "True"},
+                        },
+                    },
+                    {
+                        "name": "works_too",
+                        "external_id": "Kreistag",
+                        "conditions": [
+                            {"attribute": "kv_member_number", "condition": "KV_.*"}
+                        ],
+                        "mappings": {
+                            "comment": {
+                                "attribute": "idp_commentary",
+                                "default": "Vote weight, groups and structure levels set via SSO.",
+                            },
+                            "number": {"attribute": "participant_kv_number"},
+                            "structure_levels": {
+                                "attribute": "kv_structure",
+                                "default": "structure1",
+                            },
+                            "groups": {
+                                "attribute": "kv_group_attribute",
+                                "default": "not_a_group",
+                            },
+                            "vote_weight": {
+                                "attribute": "kv_vw",
+                                "default": "1.000000",
+                            },
+                            "present": {"attribute": "kv_presence", "default": "True"},
+                        },
+                    },
+                ],
```

Comment:
Since it seems like it can get rather long, I am wondering how will this new part be set? Programmatically via the idp, via the client by a human admin?
If it's the latter: who should set it? Only the superadmin, also the orgaadmin or should it perhaps be possible for the meetings admins to set the mapping for their meeting?
Also who should be able to access this new information?
If this new part is set by a human user other than the one who can originally set the saml_attr, we'll need to think about permissions and how to build that into the actions.
If the meeting admin should be able to set it, or (more likely I guess) see this info for his meeting, we should probably consider moving the meeting-concerned part of the mappings to the meeting models, for the sake of proper restriction in the autoupdate.
Please consider this.

Reply:
It should be that only the super admin can set the attribute mapping as a whole. As far as I know the behavior is not intended to be changed in the future.